cmd/ocp.go

```diff
@@ -66,6 +66,7 @@ func openShiftCmd() *cobra.Command {
 			return
 		}
 		util.ConfigureLogging(cmd)
+		util.SetupLogging("ocp-"+workloadConfig.UUID)
```

Comment:
You need to add surrounding spaces to "+" to pass linting rules

```suggestion
		util.SetupLogging("ocp-" + workloadConfig.UUID)
```